src/guides/v2.3/extension-dev-guide/framework/math-random.md

```diff
@@ -0,0 +1,46 @@
+---
+group: php-developer-guide
+title: Math Random
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+## Overview
+
+The [`Magento\Framework\Math\Random`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Math/Random.php){:target="\_blank"} library provides the ability to generate random values and hashes.
+
+## Usage
+
+| Method            | Description                                                                                                                                                                          |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `getRandomString` | Generates a random string with a given string length and an optional character pool. Without the optional character pool, it uses both cases of the English alphabet and all digits. |
+| `getRandomNumber` | Generates a random number with an optional range. The range defaults to 0 as minimum and the result of the PHP function `mt_getrandmax` as the maximum.                              |
+| `getUniqueHash`   | Generates a unique hash string with an optional prefix.                                                                                                                              |
+
+## Examples
+
+In the examples below, let's assume that the `$this->mathRandom` property is an instatiated object of the `Magento\Framework\Math\Random` class.
```

Comment:
`let's assume` is not a correct statement here.

`$this->mathRandom` is the instance of  `Magento\Framework\Math\Random` class

Reply:
I have indicated it as an assumption since we're actually not proving it to the reader by creating that object or injecting it in the constructor. So it's a request to the reader to _assume/imagine_ that the instantiation part is already done prior. This is my reason to use such a wording. If you feel it's still wrong, please let me know. Will change it!

Reply:
Hey, @BarnyShergold !

What is your opinion here?

Thanks

Reply:
I can see see both views, so may I suggest -

In the examples below, `$this->mathRandom` is taken to be an instantiated object of the `Magento\Framework\Math\Random` class.

I also didn't notice that "instantiated" was spelt incorrectly!

Reply:
This is good to me, @atwixfirster how about you?

Reply:
Thanks for catching that @BarnyShergold!